drizzlepac/devutils/comparison_tools/compare_sourcelists.py

```diff
@@ -356,7 +356,15 @@ def computeFlagStats(matchedRA, max_diff, plotGen, plot_title, plotfile_prefix,
                 else:
                     stat_text_blob += "\n"
             stat_text_blob += "\n" + timestamp + "\n"
-            stat_text_blob += "Comparison Sourcelist: {}\nReference Sourcelist:  {}".format(catalog_names[1], catalog_names[0])
+            if len(catalog_names[1]) > 80:
+                out_compcat_filename = "{}\n{}{}".format(catalog_names[1][:80], " "*23, catalog_names[1][80:])
+            else:
+                out_compcat_filename = catalog_names[1]
```

Comment:
This set of code is repeated (with different inputs) 4 times, making it suitable for defining a function to to reformat this string.  Something for future PRs....

Reply:
Issue resolved in commit [d4d2042](https://github.com/spacetelescope/drizzlepac/pull/660/commits/d4d204239c1aac6fa2d288df9a216f8f55b2226d)